### PR TITLE
A Pull Request for "Hello World" Project

### PR DESCRIPTION
**Pull request by YussufOB**
Hello @Omar-Muhamad this is my latest pull request and last chance for a "code review" request. The following changes were made as requested:

1.  Wrote a better descriptive README.md in business English.
2.  Added test.md file and make sure it is not present in my repo.
3.  Remove unused sections in the README.md file and include the license missing section.
4. Changed author name to mine and as well make sure all tests passed in the pull request.

Kindly help to review my code. Thanks in anticipation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
-# Hello-World
-Hello microverse project
+
+
+# Hello Microverse Project
+
+> Contains "Hello Microverse header with color styling alone.
```
Comment (OLIPLICHE) [inline README.md]: ## OPTIONAL

- [ ] Kindly write a descriptive and easy to understand `README` in business English for every project submitted. 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (OLIPLICHE) [inline README.md]: - [ ] Please remove unused sections from your README to look more professional. 👍

File: README.md [hunk 3]
```diff
@@ -1,2 +1,57 @@
...
+
+
+
+## Authors
+
+👤 **Author1**
```
Comment (OLIPLICHE) [inline README.md]: - [ ] Please Replace `Author1` with your full `name`. 👍

File: README.md [hunk 4]
```diff
@@ -1,2 +1,57 @@
...
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
```
Comment (OLIPLICHE) [inline README.md:35]: You have done amazing work✔️. Please use the suggested ReadMe [template](https://github.com/microverseinc/readme-template) to add the missing section in your Readme ( `License`).

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @YussufOB , 

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!

## Highlights
Excellent Job 👏 👏👏

Linter is set up Correctly ✔️
Your Linter result is green ✔️
Github flow is correctly used ✔️
Your commit messages are meaningful✔️
There is a .gitignore file listing `node_modules` directory✔️


## Required Changes ♻️
- Please add  `test.md` file with the text "This file should be ignored by git", and make sure that the test.md file is ignored by git and is not present in your GitHub repository.
- Please add a descriptive title in your Pull Request, telling you what changes are inside.
- Please your pull request should have a summary describing introduced changes in more detail
- CHECK THE COMMENTS UNDER THE REVIEW ⬇.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (YussufOB) [reply README.md:35]: Thanks for your time to review my code. No section is missed except not needed. However, if there is any (maybe by an error of commission), please help to highlight the section.

**Comment by YussufOB** [comment]
Hello @OLIPLICHE, this is my latest pull request. The following changes were made as requested:

1. Wrote a better descriptive README.md in business English.
2. Added test.md file.
3. Remove unused section in the README.md file.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @YussufOB 

Nice job making the required changes. Unfortunately, you missed some of them but you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. :heavy_check_mark:
- All linters checks are ok. :heavy_check_mark:

## (Changes Required) Aspects to improve: ♻️

- [ ] Please add `license` section like the last reviewer requested.

image

- [ ] Pull Request should have a summary describing introduced changes in more detail so please change it.

image

- [ ] Kindly remove `test.md` from your repo and please remember next to add any file you want to be ignored to the `.gitignore` file before you push any changes.

image


## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AnselemOdims — APPROVED** [review]
### Hi  @YussufOB 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**